### DISCUSSION TITLE
How can I move weather/widgets to the left on the DockLike theme for Taskbar Styler?

### DISCUSSION THREAD

**Discussion by klydeinside** [ramensoftware/windhawk-mods#942] [Mod ideas]
Is there anyone that can help me move the weather/widgets part of the taskbar to the very left side? I like the start stuff in the middle, the notifications on the right, and would be nice if I could get the weather on the left side so everything is spread out somewhat evenely. If anyone could help, that would be greatly appreciated.

**Comment by m417z** [comment]
It's best to ask here and tag the theme author:
https://github.com/ramensoftware/windows-11-taskbar-styling-guide
